src/js/movie-list.js

```diff
@@ -0,0 +1,25 @@
+export const createCard = ({ backdrop_path, title, genre, release_date }) => {
```

Comment:
zmerguje to i zaraz będę cykac w poglądzie na żywo czy działają te js, i to jest poza fetch-api i zadziała tu, czy musiałoby być w fetch-api?